packages/nodes-base/nodes/MessageBird/MessageBird.node.ts

```diff
@@ -0,0 +1,336 @@
+import { IExecuteFunctions } from 'n8n-core';
+import {
+	IDataObject,
+	INodeTypeDescription,
+	INodeExecutionData,
+	INodeType
+} from 'n8n-workflow';
+
+import { messageBirdApiRequest } from './GenericFunctions';
+
+export class MessageBird implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'MessageBird',
+		name: 'messageBird',
+		icon: 'file:messagebird.png',
+		group: ['transform'],
+		version: 1,
+		subtitle: '={{$parameter["operation"] + ": " + $parameter["resource"]}}',
+		description: 'Sends SMS',
```

Comment:
Also offers chat and voice functionality, but only SMS implemented for now. I'm not sure if this field is supposed to describe all operations or just the ones implemented. The docs don't specify.